blocks/cards/cards.css

```diff
@@ -155,4 +155,59 @@
   .cards.profile-summary > ul > li img {
     border-radius: 8px 0 0 8px;
   }
+}
+
+/* Senior Leadership View Styles */
+
+.cards.senior-leadership {
+  padding: 0;
+}
+
+.cards.senior-leadership > ul {
+  grid-template-columns: 1fr 1fr;
+  grid-gap: 1.5rem;
+}
+
+.cards.senior-leadership > ul > li {
+  border: none
+}
+
+.cards.senior-leadership .cards-card-image img {
+  border-radius: 5px;
+}
+
+.cards.senior-leadership .cards-card-body {
+  margin: 1rem 0 0;
+}
+
+.cards.senior-leadership .cards-card-body h4 {
+  margin-bottom: 0.6rem;
```

Comment:
for bottom and top margin's you can use `em's`. Or `px` is also fine.

Reply:
done